glide.yaml

```diff
@@ -1,25 +1,14 @@
 package: sigs.k8s.io/descheduler
 import:
 - package: k8s.io/client-go
-  version: 78700dec6369ba22221b72770783300f143df150
+  version: kubernetes-1.16.0
```

Comment:
@Huang-Wei Does this version also include the even spread api or is that in some alpha branch ?

Reply:
Yes.